manager/src/main/java/com/redhat/service/smartevents/manager/api/user/SchemaAPI.java

```diff
@@ -0,0 +1,82 @@
+package com.redhat.service.smartevents.manager.api.user;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.annotation.PostConstruct;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.apache.commons.io.IOUtils;
+import org.eclipse.microprofile.openapi.annotations.Operation;
+import org.eclipse.microprofile.openapi.annotations.enums.SecuritySchemeType;
+import org.eclipse.microprofile.openapi.annotations.media.Content;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponses;
+import org.eclipse.microprofile.openapi.annotations.security.SecurityRequirement;
+import org.eclipse.microprofile.openapi.annotations.security.SecurityScheme;
+import org.eclipse.microprofile.openapi.annotations.security.SecuritySchemes;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+
+import com.redhat.service.smartevents.infra.api.APIConstants;
+import com.redhat.service.smartevents.manager.api.models.responses.ProcessorCatalogResponse;
+import com.redhat.service.smartevents.manager.api.models.responses.ProcessorSchemaEntryResponse;
+
+import io.quarkus.security.Authenticated;
+
+@Tag(name = "Schema Catalog", description = "The API that provide the JSON schemas for the actions/sources processors")
```

Comment:
I think this endpoint provides the general catalogue of the supported action/sources, rather than only the Json Schema

Reply:
The tag is for the section in the OpenAPI document: this section is going to contain the catalog endpoint and the json schema endpoint